api/src/main/java/org/pmiops/workbench/api/CloudTaskUserController.java

```diff
@@ -135,11 +134,11 @@ public ResponseEntity<Void> synchronizeUserAccess(SynchronizeUserAccessRequest r
           }
         }
 
-        // Always synchronize for consistency. Under normal operation only the passage of time
-        // should result in access expiration changes here, but this is also a backstop for ensuring
-        // the database is consistent with our access tier groups, e.g. due to partial system
-        // failures or bugs.
-        userService.updateUserAccessTiers(user, Agent.asSystem());
```

Comment:
no longer needed, because syncDuccVersionStatus() covers this.  See comment for more details, and let me know if something is unclear.